# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1391026677 18000
#      Wed Jan 29 15:17:57 2014 -0500
# Branch stable
# Node ID 61d9ef05b06bee5340bf916e4fa16a35fb5c9a25
# Parent  a4b4cf4c6ece3f62c470bbfe5f3690ea3188056d
consistently use '...' without preceding space in GUI messaages

diff --git a/libgui/languages/be_BY.ts b/libgui/languages/be_BY.ts
--- a/libgui/languages/be_BY.ts
+++ b/libgui/languages/be_BY.ts
@@ -893,17 +893,17 @@ Do you want to delete it anyway?</source
     </message>
     <message>
         <location line="+1"/>
         <source>Write %1 file</source>
         <translation>Запісаць %1-файл</translation>
     </message>
     <message>
         <location line="+71"/>
-        <source>Print To File ...</source>
+        <source>Print To File...</source>
         <translation>Друк у файл...</translation>
     </message>
     <message>
         <location line="+78"/>
         <source>%1 is a directory.
 Please choose a different file name.</source>
         <translation>%1 - гэта каталог.
 Калі ласка, пазначце іншую назву файла.</translation>
@@ -2033,17 +2033,17 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Дзеянні з дзейным каталогам</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation>Шукаць файлы...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Новы файл</translation>
     </message>
     <message>
diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -493,17 +493,17 @@ wurde gelöscht oder umbenannt. Soll die Datei jetzt gespeichert werden?%2</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Aktionen mit aktuellem Verzeichnis</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation>Dateien suchen...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Neue Datei</translation>
     </message>
     <message>
@@ -881,17 +881,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+5"/>
         <source>Enter the start directory</source>
         <translation>Verzeichnis, in dem die Suche beginnt</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse...</source>
-        <translation>Suche ...</translation>
+        <translation>Suche...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse for start directory</source>
         <translation>Suche Startverzeichnis</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -986,17 +986,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+17"/>
         <source>File contents</source>
         <translation>Dateiinhalt</translation>
     </message>
     <message>
         <location line="+105"/>
         <source>Searching...</source>
-        <translation>Suche ...</translation>
+        <translation>Suche...</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Set search directory</source>
         <translation>Suchverzeichnis setzen</translation>
     </message>
 </context>
 <context>
diff --git a/libgui/languages/en_US.ts b/libgui/languages/en_US.ts
--- a/libgui/languages/en_US.ts
+++ b/libgui/languages/en_US.ts
@@ -476,17 +476,17 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -476,18 +476,18 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Acciones en el directorio actual</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
-        <translation>Buscar archivos ...</translation>
+        <source>Find Files...</source>
+        <translation>Buscar archivos...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Nuevo archivo</translation>
     </message>
     <message>
         <location line="+3"/>
diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -492,17 +492,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Actions sur le répertoire courant</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation>Rechercher des fichiers...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Nouveau fichier</translation>
     </message>
     <message>
diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -892,17 +892,17 @@ Wil je die toch verwijderen?</translatio
     </message>
     <message>
         <location line="+1"/>
         <source>Write %1 file</source>
         <translation>Bewaar %1 bestand</translation>
     </message>
     <message>
         <location line="+71"/>
-        <source>Print To File ...</source>
+        <source>Print To File...</source>
         <translation>Druk af naar bestand...</translation>
     </message>
     <message>
         <location line="+78"/>
         <source>%1 is a directory.
 Please choose a different file name.</source>
         <translation>%1 is een map.
 Kies s.v.p. een andere bestandsnaam.</translation>
@@ -2031,18 +2031,18 @@ is gewist of hernoemd. Wil je het nu ops
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Bewerkingen op huidige map</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
-        <translation>Zoek bestanden ...</translation>
+        <source>Find Files...</source>
+        <translation>Zoek bestanden...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Nieuw bestand</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -2420,17 +2420,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+5"/>
         <source>Enter the start directory</source>
         <translation>Geef startmap op</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse...</source>
-        <translation>Bladeren ...</translation>
+        <translation>Bladeren...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse for start directory</source>
         <translation>Blader naar begin map</translation>
     </message>
     <message>
         <location line="+3"/>
diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_BR.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_BR.ts
@@ -490,17 +490,17 @@ foi removido ou renomeado. Gostaria de salvá-lo agora?%2</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Ações no diretório atual</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation>Buscar Arquivos...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Novo Arquivo</translation>
     </message>
     <message>
diff --git a/libgui/languages/pt_PT.ts b/libgui/languages/pt_PT.ts
--- a/libgui/languages/pt_PT.ts
+++ b/libgui/languages/pt_PT.ts
@@ -1159,17 +1159,17 @@ foi removido ou renomeado. Pretende guardá-lo agora?%2</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Acções no directório actual</translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation>Procurar Ficheiros...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Novo Ficheiro</translation>
     </message>
     <message>
diff --git a/libgui/languages/ru_RU.ts b/libgui/languages/ru_RU.ts
--- a/libgui/languages/ru_RU.ts
+++ b/libgui/languages/ru_RU.ts
@@ -492,17 +492,17 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation></translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation>Найти файлы...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Новый файл</translation>
     </message>
     <message>
diff --git a/libgui/languages/uk_UA.ts b/libgui/languages/uk_UA.ts
--- a/libgui/languages/uk_UA.ts
+++ b/libgui/languages/uk_UA.ts
@@ -1142,17 +1142,17 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+373"/>
-        <source>Find Files ...</source>
+        <source>Find Files...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/libgui/src/files-dock-widget.cc b/libgui/src/files-dock-widget.cc
--- a/libgui/src/files-dock-widget.cc
+++ b/libgui/src/files-dock-widget.cc
@@ -492,17 +492,17 @@ files_dock_widget::contextmenu_requested
       if (info.isDir ())
         {
           menu.addSeparator ();
           menu.addAction (QIcon (":/actions/icons/ok.png"),
                           tr ("Set Current Directory"),
                           this, SLOT (contextmenu_setcurrentdir (bool)));
           menu.addSeparator ();
           menu.addAction (QIcon (":/actions/icons/findf.png"),
-                          tr ("Find Files ..."), this,
+                          tr ("Find Files..."), this,
                           SLOT (contextmenu_findfiles (bool)));
         }
 
       menu.addSeparator ();
       menu.addAction (tr ("Rename"), this, SLOT (contextmenu_rename (bool)));
       menu.addAction (QIcon (":/actions/icons/editdelete.png"), tr ("Delete"),
                       this, SLOT (contextmenu_delete (bool)));
 
